### PR TITLE
Check Employee IP address in BO

### LINKED ISSUE #36243: Check IP when connected on BO
Reimplement PS_COOKIE_CHECKIP functionality removed in https://github.com/PrestaShop/PrestaShop/pull/35983. This feature allows to enable/disable checking IP address of the session. Helps prevent cookie theft. Some people running proxies disabled it, because the IP behind the proxy fluctuated and they were logged out every few minutes. Good would be to check both client IP address and X-Forwarded-For header.

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Save employee IP address in token attributes to check it on each further request
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green UI tests green (For functional test you need to setup a website accessible online and use a VPN to switch IP, see with a developer for this test)
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/9272864550
| Fixed issue or discussion?     | Fixes #36243
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

**Review by boherm — DISMISSED** [review]

File: src/PrestaShopBundle/EventListener/Admin/EmployeeSessionSubscriber.php
Comment (nicosomb) [inline src/PrestaShopBundle/EventListener/Admin/EmployeeSessionSubscriber.php]:
```suggestion
            $session->getFlashBag()->add('warning', $this->translator->trans('You have been logged out for security reasons', [], 'Admin.Login.Feature'));
```

?

**Review by nicosomb — DISMISSED** [review]
ok for me, just a non mandatory suggestion.

**Comment by ps-jarvis** [comment]
This pull request seems to contain new translation strings. I have summarized them below to ease up review:

- `Admin.Login.Feature`
    - [ ] `You have been logged out for security reasons`

(Note: this is an automated message, but answering it will reach a real human)

**Comment by tleon** [comment]
I did a QA by dev on this it's working fine for me

**Review by tleon — APPROVED** [review]

**Comment by ps-jarvis** [comment]
QA approved, well done! Message to the maintainers: do not forget to milestone it before the merge.

**Review by nicosomb — APPROVED** [review]

**Pull request merged**
